### PR TITLE
Add PA Dept of Revenue

### PR DESCRIPTION
**Pull request by mmomjian**
No docs available, however see [press release](https://www.revenue.pa.gov/OnlineServices/mypath/Documents/myPATH_V12_updates.pdf) for confirmation of TOTP, SMS, and Email support. I have tested it myself.

### CODE REVIEW

**Comment by Carlgo11** [comment]
Thanks for the pull request @mmomjian!

I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

File: _data/government.yml [hunk 1]
```diff
@@ -140,6 +140,16 @@ websites:
     regions:
       - au
 
+  - name: Pennsylvania Dept of Revenue
+    url: https://mypath.pa.gov
+    img: PA-revenue.png
```
Comment (Carlgo11) [inline _data/government.yml:145]:
```suggestion
    img: mypath.png
```
The image names are normally all lowercase. Perhapt mypath.png works better?

Comment (mmomjian) [reply _data/government.yml:145]: I didn't know they were all lowercase, I see the Jekyll regex failed. mypath should work fine as well

Comment (mmomjian) [reply _data/government.yml:145]: Do I need to re upload the image or can you fix it?

Comment (Carlgo11) [reply _data/government.yml:145]: I'm on my phone right now so I'm afraid I can't change that at the moment.
If you'd like to remove rename the image that'd be great. If not, I'll do it in a couple of hours 🙂

**Comment by mmomjian** [comment]
Will rename files

**Pull request closed without merging**